src/system.cpp

```diff
@@ -1047,6 +1047,23 @@ bool System::command_customizations(const char * value, const int8_t id, JsonObj
             sensorJson["offset"]  = sensor.offset; // o
         }
 
+        JsonArray analogJson = node.createNestedArray("analogs");
+        for (const AnalogCustomization & sensor : settings.analogCustomizations) {
+            JsonObject sensorJson = analogJson.createNestedObject();
+            sensorJson["gpio"]    = sensor.id;
+            sensorJson["name"]    = sensor.name;
+            if (sensor.type == AnalogSensor::AnalogType::ADC) {
+                sensorJson["type"]   = "adc";
```

Comment:
why switch from the enum to strings?

Reply:
You're right, better to check the enum-format and put the strings to a flash-list.